src/controller/python/matter/commissioning/commissioning_flow_blocks.py

```diff
@@ -65,15 +70,15 @@ async def operational_credentials_commissioning(self, parameter: commissioning.P
         # Failures are exceptions
         try:
             dac = await self._devCtrl.SendCommand(node_id, commissioning.ROOT_ENDPOINT_ID, Clusters.OperationalCredentials.Commands.CertificateChainRequest(
-                certificateType=1
+                certificateType=Clusters.OperationalCredentials.Enums.CertificateChainTypeEnum(1)
```

Comment:
If you're doing this already, can you use the proper enum values here? kDACCertificate